core/CMakeLists.txt

```diff
@@ -65,15 +58,15 @@ target_link_libraries(rwlib
         OpenGL::GL
     )
 
-set_target_properties(rwlib
+set_target_properties(core
         PROPERTIES
             OUTPUT_NAME rw
```

Comment:
These lines can be removed. The output name `libcore`  (or `librwcore`) is fine. (Where librwlib was weird)